Makefile

```diff
@@ -2,10 +2,7 @@ all: build
 
 .PHONY: build
 
-ifndef $GOPATH
-	GOPATH = $(HOME)/go
-endif
-
+GOPATH ?= $(HOME)/go
```

Comment:
This was interesting.

I don't know if it's always been broken, and only just became an issue. Or if it's never worked worked correctly. But looking at the old code, the intent was clearly to set GOPATH if it was not already set, and then to add that to PATH. I think this is because we install a handful of tooling/ci dependancies, and they need to be in the PATH to work.

But, the old logic wasn't working. In CI. GOPATH was blank, the GOPATH setting didn't take, and then we would prepend `/bin` to path. On the CI machine, go 1.17 was in `/bin`, which caused the build to fail. (Since I started using `any`)